code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -160,11 +161,14 @@
 	icon_state = "secglock[chambered ? "" : "-e"][silenced ? "-s" : ""][stored_magazine ? "" : "-m"][clowned == CLOWNED ? "-c" : ""]"
 
 /obj/item/weapon/gun/projectile/glock/fancy
-	desc = "The NT Glock is a cheap, ubiquitous sidearm, produced by a NanoTrasen subsidiary. Uses .380AUTO rounds.<br><span class='notice'>This one has a sweet platinum-plated slide, and tritium night sights for maint crawling!</span>"
 	name = "\improper NT Glock Custom"
 	icon_state = "secgunfancy"
 	clowned = UNCLOWN
 
+/obj/item/weapon/gun/projectile/glock/fancy/New()
+	. = ..()
+	desc += "<br><span class='notice'>This one has a sweet platinum-plated slide, and tritium night sights for maint crawling!</span>"
```

Comment:
Should replace maint with maintenance, you know, just for the looks

Reply:
it is done